conda_smithy/templates/run_docker_build.tmpl

```diff
@@ -8,21 +8,16 @@
 FEEDSTOCK_ROOT=$(cd "$(dirname "$0")/.."; pwd;)
 RECIPE_ROOT=$FEEDSTOCK_ROOT/{{ recipe_dir }}
 
-docker info
-
 config=$(cat <<CONDARC
 
 channels:
 {%- for channel in channels.get('sources', []) | sort %}
  - {{ channel }}
 {% endfor %}
- - defaults # As we need conda-build
```

Comment:
When I remove `defaults` from my (windows) condarc, I can't get a python package. In the other setups, it always "adds" to the channel list, so I would guess that default should stay here? Or is that different on other OSes?